includes/abstracts/abstract-wc-stripe-payment-gateway.php

```
@@ -145,10 +145,16 @@ public function save_payment_method_checkbox( $force_checked = false ) {
 	 * @param object $error
 	 */
 	public function is_retryable_error( $error ) {
-		// We don't want to retry payments when a 3DS mandate is invalid; it doesn't make sense.
 		// Note that this check is required since the error type is 'invalid_request_error' which
 		// would otherwise return true.
-		if ( isset( $error->code ) && 'payment_intent_mandate_invalid' === $error->code ) {
+		if (
+			isset( $error->code ) &&
+			(
+				'payment_intent_mandate_invalid' === $error->code || // Don't retry payments when a 3DS mandate is invalid.
+				'charge_exceeds_transaction_limit' === $error->code || // Don't retry payments when the charge exceeds the transaction limit.
+				'amount_too_small' === $error->code
+			)
+		) {
```

FYI @asumaran @rafaelzaleski – In case you find any issues with retrials due to ACH/BACS specific errors, you can check and ignore the error in `is_retryable_error`.